packages/core/docs/utilities.stories.ts

```diff
@@ -49,14 +50,17 @@ export const internationalization = () => {
 export const lazyLoading = () => {
   class LazyDemo extends LitElement {
     @property({ type: Boolean })
-    loaded = false;
+    loaded = true;
```

Comment:
I guess I'm missing why this was changed. It seems now that `loaded` is always true. So don't we lose the lazyLoading aspect of this story now?

Reply:
ah good catch, the dropdown file changes shouldnt be in the PR. I was using it for a test

Reply:
Fixed, sorry for the force push. I messed up my rebase earlier. That file should be removed now.